frontend/vue-gewoscout/src/components/SearchResultList.vue

```diff
@@ -0,0 +1,67 @@
+<script lang="ts" setup>
+import { onMounted, ref, type Ref } from 'vue';
+import Button from 'primevue/button';
+import Card from 'primevue/card';
+import { getListings } from '@/common/api-service';
+import type { Listing } from '@/types/ApiResponseListings';
+
+const props = withDefaults(defineProps<{ searchCity?: string }>(), {
+  searchCity: 'vienna'
+});
+
+const results: Ref<Listing[]> = ref([]);
+onMounted(async () => {
+  results.value = await getListings(props.searchCity);
+});
+
+function redirectToApartment(index: number) {
+  window.open(results.value[index].detailsUrl, '_blank');
+}
+</script>
+
+<template>
+  <div class="cards mt-3 grid">
+    <div class="col-12 lg:col-4" v-for="(item, index) in results" :key="index">
```

Comment:
I have updated this section and moved the dummy data to a dummy API server. These API calls should work as they would in production.